src/tests/core/engine/processors/kitchen_simulator_test.py

```diff
@@ -0,0 +1,28 @@
+from src.core.engine.processors.kitchen_simulator import KitchenSimulator
+from src.tests.utils.fixtures.app_processor_config_fixture import (
+    build_app_processor_config,
+)
+from src.tests.utils.fixtures.chef_fixture import build_chef
+from src.tests.utils.fixtures.order_fixture import build_order
+from src.constants.order_status import OrderStatus
+import unittest
+
+
+class KitchenSimulatorTest(unittest.TestCase):
+    def test_assign_order_to_available_chef(self):
```

Comment:
This look like it is an integration test. Unit tests should mock all the controllers.

Reply:
Also you should verify if these controllers were called as expected

Reply:
done